pos_customer_wallet/tests/common.py

```diff
@@ -9,6 +9,7 @@ class TestPosBalance(TestBalance):
     @classmethod
     def setUpClass(cls, *args, **kwargs):
         super().setUpClass(*args, **kwargs)
+        cls.PosOrder = cls.env["pos.order"]
```

Comment:
Why do we do this?

Reply:
no real reason. maybe some OCA convention, but not sure. I removed it.